vpn/tunnel.go

```diff
@@ -151,10 +165,11 @@ func (t *Tunnel) handleRPC(req *ManagerMessage, msgID uint64) *TunnelMessage {
 				ErrorMessage: errStr,
 			},
 		}
-		return resp
 	default:
 		t.logger.Warn(t.ctx, "unhandled manager request", slog.F("request", msg))
-		return resp
+	}
+	if err := req.sendReply(resp); err != nil {
```

Comment:
The original code also sent `resp := &TunnelMessage{}` as a reply for the `default:` branch, so I assume it's still fine.